### DISCUSSION TITLE
Grouping

### DISCUSSION THREAD

**Discussion by dcblogdev** [spatie/ray#214] [Ideas]
All plans for a way to group the errors per project? like how HELO works:

<img width="373" alt="Screenshot 2021-01-19 at 20 56 25" src="https://user-images.githubusercontent.com/1018170/105092285-f2cd4600-5a98-11eb-98c4-61a5421a5871.png">

Would be nice to be able to toggle over projects

**Comment by freekmurze** [comment]
In most cases I'm only debugging one project at a time, so I'm hesitant of adding something like this.

I might change my stance if more people ask for this.

**Reply by vesper8** [reply]
@freekmurze I need this grouping so badly... I've set up  Ray so that production projects all output to ray via an always-running `expose` tunnel. So I have input coming in from many apps, both production and dev.. to a single Ray instance. This makes the need for some kind of tabs/grouping system that much greater

I hope you will add this soon, but if not then at least if there was a way to expand on the colour system.. to have more colours.. more dots.. maybe I could have different apps use different colours

**Reply by freekmurze** [reply]
Probably we won't do this quickly. Ray's primary use case is to help you debug something and in that context the correct number of colours is enough.

It's pretty cool though that you want to use it for something it's not primarily intended for.
